.circleci/config.yml

```diff
@@ -834,12 +833,12 @@ jobs:
           # Uses --debug for compilation speed
           # FIXME: Change when `check_contract` (part of `cosmwasm-0.14.0`) is published
           #command: cargo install --debug --features iterator --example check_contract -- cosmwasm-vm
-          command: cargo install --debug --features iterator --git https://github.com/CosmWasm/cosmwasm --tag=v0.14.0-beta1 --example check_contract -- cosmwasm-vm
+          command: cargo install --debug --features iterator --git https://github.com/CosmWasm/cosmwasm --branch=main --example check_contract -- cosmwasm-vm
```

Comment:
I wouldn't make this change here. Rather keep it on a tag (can bump to -beta2 when ready)

Reply:
This should in principle be changed to a released version (i.e. the commented line above). I'm OK with disabling the main version, though.